docs/controls/Line.md

```diff
@@ -203,8 +203,9 @@ Each line can have a legendOptions object in [Values](#values) level.
 
 | Property Name | Expected | Default | Description                                           |
 | ------------- | -------- | ------- | ----------------------------------------------------- |
-| showShape     | boolean  | true    | Display Shape in the legend.                          |
+| showElement   | boolean  | true    | Toggle to hide legend when legend item has no data.   |
```

Comment:
if the consumer already has the ```legendOptions``` object and doesn't have the ```showElement``` prop in their repo, then legend will display as did previously. If the legend is not holding any data, then it will be disabled by default.

Reply:
Then should it be `showElement` prop is optional?

Reply:
Updated [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/61c9833d8058dfdce37354a2c719627d9b1c3cfb)

Reply:
I think all the properties should be optional in the legend options!

Reply:
All properties [moved](https://github.com/NarasimhaShenoi/carbon-graphs/commit/cd4edefe5fb917e528894c96dd60d7a29616f179) to optional in legend options

Reply:
+1